CMSgov/easi-app
# [NOREF] Upgrade to go 1.18.3

# NOREF

## Changes and Description

- Upgrades the go version of the project from 1.16.6 to 1.18.3

## How to test this change

- Upgrade your go version locally to go 1.18.3
  - How this is done will vary depending on how you installed it!
- Ensure that backend tests run with `scripts/dev up:backend` followed by `scripts/dev test:go`
- Ensure that frontend tests run by following instructions in [local_testing.md](https://github.com/CMSgov/easi-app/blob/6ea5543b810bbefec5ea15d663f351e87147a69d/docs/local_testing.md#L20-L36)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
